### DISCUSSION TITLE
Hi, does it soppurt touchscreen?

### DISCUSSION THREAD

**Discussion by danistmein** [bulletmark/libinput-gestures#341] [Q&A] [answered]
Hi, does it support touchscreen?

**Comment by bulletmark** [comment, marked as answer]
No sorry, it only supports what libinput calls gestures which are touchpad only. See opening paragraph [here](https://wayland.freedesktop.org/libinput/doc/latest/gestures.html#gestures).

**Reply by danistmein** [reply]
Thanks,
Best Regards!
